posthog-ai/package.json

```diff
@@ -34,6 +34,9 @@
     "uuid": "^11.0.5",
     "zod": "^3.24.1"
   },
+  "peerDependencies": {
+    "posthog-node": "^4.17.1"
+  },
```

Comment:
@k11kirky this is probably the most relevant part. Just wondering if there is any reason you didn't previously do this?